addon/components/bourbon-search-select-field.js

```diff
@@ -101,6 +113,17 @@ export default Component.extend({
     }
   },
 
+  resetPrompt: observer("label", function () {
+    if (this.get("label")) {
+      this.set("inputValue", this.get("label"));
+    } else if (
+      this.get("prompt") &&
```

Comment:
This can probably all be on one line. I get wanting to spread things out for readability, but this isn't that long to be unreadable.